xsoar/data_source_integration_instance.go

```diff
@@ -112,7 +113,7 @@ func (r dataSourceIntegrationInstance) Read(ctx context.Context, req tfsdk.ReadD
 		Account:           config.Account,
 		PropagationLabels: types.List{Elems: propagationLabels, ElemType: types.StringType},
 		Config:            config.Config,
-		IncomingMapperId:  config.IncomingMapperId,
+		IncomingMapperId:  types.String{Value: integration["incomingMapperId"].(string)},
```

Comment:
This has the potential to panic if the value of the `interface{}` is `nil`. When using type conversion in `nil` it's suggested to use multiple assignment and verify the result.

```go
incomingMapperId, ok := integration["incomingMapperId"].(string)
if ok {
    result.incomingMapperId = types.String{Value: incomingMapperId}
} else {
    result.incomingMapperId = types.String{Null: true}
}
```